AppCenterDataStorage/AppCenterDataStorage/Internal/Model/MSPaginatedDocumentsInternal.h

```diff
@@ -4,15 +4,19 @@
 #import "MSPage.h"
 #import "MSReadOptions.h"
 
+NS_ASSUME_NONNULL_BEGIN
+
 @interface MSPaginatedDocuments ()
 
 // Read-only.
-@property(nonatomic, copy, readonly) NSString *partition;
-@property(nonatomic, readonly) Class documentType;
-@property(nonatomic, readonly) MSReadOptions *readOptions;
+@property(nonatomic, copy, readonly, nullable) NSString *partition;
+@property(nonatomic, readonly, nullable) Class documentType;
```

Comment:
This one also should be non nullable. The change will need to be propagated.
```suggestion
@property(nonatomic, readonly) Class documentType;
```